### PR TITLE
[Translation] LoggingTranslator simplifications

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by stof** [comment]
:+1:

**Comment by Tobion** [comment]
:+1:

File: src/Symfony/Component/Translation/LoggingTranslator.php [hunk 1]
```diff
@@ -29,12 +29,12 @@ class LoggingTranslator implements TranslatorInterface, TranslatorBagInterface
...
-     * @param LoggerInterface $logger
+     * @param TranslatorInterface $translator The translator must implement TranslatorBagInterface
+     * @param LoggerInterface     $logger
      */
-    public function __construct($translator, LoggerInterface $logger)
+    public function __construct(TranslatorInterface $translator, LoggerInterface $logger)
     {
-        if (!($translator instanceof TranslatorInterface && $translator instanceof TranslatorBagInterface)) {
+        if (!$translator instanceof TranslatorBagInterface) {
             throw new \InvalidArgumentException(sprintf('The Translator "%s" must implements TranslatorInterface and TranslatorBagInterface.', get_class($translator)));
```
Comment (Tobion) [inline src/Symfony/Component/Translation/LoggingTranslator.php]: typo: must implement

Comment (xabbuh) [inline src/Symfony/Component/Translation/LoggingTranslator.php]: fixed

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**